File: 백준/Bronze/2581. 소수/소수.cc
#include <iostream>
#include <vector>
using namespace std;

int main(void) {
	ios_base::sync_with_stdio(false);
	cin.tie(nullptr);

	int m = 0, n = 0;
	if (!(cin >> m >> n)) return 0;

	int dsum = 0;
	int dmin = 10001;
	for (int i = m; i <= n; i++) {
		if (i < 2) continue;

		bool isdecimal = true;
		for (int j = 2; j < i; j++) {
			if (i % j == 0) {
				isdecimal = false;
				break;
			}
		}

		if (isdecimal) {
			dsum += i;
			if (dmin > i) {
				dmin = i;
			}
		}
	}

	if (dsum == 0) {
		cout << -1 << "\n";
	}
	else {
		cout << dsum << "\n" << dmin << "\n";
	}

	return 0;
}

File: 백준/Bronze/2581. 소수/소수.py
import sys
input = sys.stdin.read

def solve():
    data = input().split()
    if not data: return
    m, n = map(int, data)
    
    dsum = 0
    dmin = 10001
    for i in range(m, n + 1):
        if i < 2: continue
        
        isdecimal = True
        for j in range(2, i):
            if i % j == 0:
                isdecimal = False
                break
        
        if isdecimal: 
            dsum += i
            if dmin > i:
                dmin = i
                
    if dsum == 0:
        print(-1)
    else:
        print(dsum, dmin, sep="\n")
            
if __name__ == "__main__":
    solve()